src/main/java/com/revature/controllers/PersonController.java

```diff
@@ -155,14 +155,14 @@ else if(role!=0 && "".equals(roleName)){
 	}
 	
 	@RequestMapping(method = RequestMethod.DELETE, value = "persons/{pId}")
-	public ResponseEntity<String> deletePerson(@PathVariable("pId") Integer pId){
+	public ResponseEntity<Person> deletePerson(@PathVariable("pId") Integer pId){
 	
 			Person pers = personLogic.getPersonById(pId);
 			
 			if (pers != null) {
 				personLogic.deletePerson(pers);
-				String message = pers.getFirstName() + " " + pers.getLastName() + " was deleted.";
-				return ResponseEntity.ok(message);
+//				String message = pers.getFirstName() + " " + pers.getLastName() + " was deleted.";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)